metron-interface/metron-config/angular-cli.json

```diff
@@ -0,0 +1,51 @@
+{
+  "project": {
+    "version": "1.0.0-beta.15",
```

Comment:
Should we version this to match Metron versions? It's likely to run lock step with Metron from a release perspective.